### ISSUE TITLE
Use AWS IAM Access Analyzer in AWS Organizations

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#12593]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Analyzers can now be created in the AWS Organizations master account or a delegated member account with the entire organization as the zone of trust.

### New or Affected Resource(s)

* [`aws_accessanalyzer_analyzer`](https://www.terraform.io/docs/providers/aws/r/accessanalyzer_analyzer.html)

### Potential Terraform Configuration

```hcl
resource "aws_accessanalyzer_analyzer" "example" {
  type = "ORGANIZATION"
}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2020/03/review-and-remediate-unintended-access-allowed-on-your-aws-resources-from-outside-your-aws-organization/).
[Blog post](https://aws.amazon.com/blogs/aws/new-use-aws-iam-access-analyzer-in-aws-organizations/).

Requires [AWS SDK v1.30.0](https://github.com/aws/aws-sdk-go/releases/tag/v1.30.0):
* https://github.com/terraform-providers/terraform-provider-aws/pull/12414

Related:
* https://github.com/terraform-providers/terraform-provider-aws/issues/11102
* https://github.com/terraform-providers/terraform-provider-aws/pull/11169

**Comment by seanliu2fox** [comment]
Organizations definitely needs to supported for the analyzer.

**Comment by sutharshan-sharma** [comment]
@ewbankkit @bflad Any example usage available for aws_accessanalyzer_archive_rule resource in Terraform ?

**Comment by diego-ojeda-binbash** [comment]
I would also appreciate that this is implemented 🙏 

Its usage should be straightforward:
```
resource "aws_accessanalyzer_analyzer" "example" {
  name = "example"
  type = "ORGANIZATION"
}
```

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 3.2.0 of the Terraform AWS Provider, later this week. 👍
